src/api-serverless/src/nft-owners/api.nft-owners.routes.ts

```diff
@@ -1,13 +1,49 @@
 import { Request } from 'express';
 import { asyncRouter } from '../async.router';
 import { DEFAULT_PAGE_SIZE } from '../page-request';
-import { returnPaginatedResult } from '../api-helpers';
-import { fetchNftOwnersForConsolidation } from './api.nft-owners.db';
+import { resolveSortDirection, returnPaginatedResult } from '../api-helpers';
+import {
+  fetchAllNftOwners,
+  fetchNftOwnersForConsolidation
+} from './api.nft-owners.db';
 
 const router = asyncRouter();
 
 export default router;
 
+router.get(
+  '/',
+  async function (
+    req: Request<
+      any,
+      any,
+      any,
+      {
+        contract?: string;
+        token_id?: string;
+        sort_direction: any;
+        page?: number;
+        page_size?: number;
+      }
+    >,
+    res: any
```

Comment:
Since you do that, I can go and change anything in openapi.yaml in NftOwner and nothing will break but the API will go out of sync with the documentation